controllers/front/CartController.php

```diff
@@ -238,7 +238,7 @@ public function postProcess()
     protected function updateCart()
     {
         // Update the cart ONLY if $this->cookies are available, in order to avoid ghost carts created by bots
```

Comment:
Maybe we should just change, like this, to comment why this check was there
```
// Update the cart ONLY if it's not a bot, in order to avoid ghost carts
```

Reply:
@matks ping !

Reply:
Je viens de mettre à jour le commentaire